src/App.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "components/weather";
+import { shallow } from 'enzyme';
+import { fetchWeather } from 'api/weather';
+
+
+describe('React component test with Enzyme', () => {
+  it('renders without crashing', () => {
+     shallow(<App />);
+   });
+});
```

Comment:
+100 for tests! Could you go over a few tests you'd want to run if this were a real app?